README.md

```diff
@@ -590,6 +584,8 @@ This saves time in projects where builds can stay long time in a build queue and
 
 ## Compatibility
 
+During the migration of the GitLab Plugin from RESTEasy library to GitLab4J-API as GitLab has dropped support for API V3, the support for GitLab API V3 has also been removed from the plugin and only the latest V4 is supported.
+
```

Comment:
I don't see how this has anything to do with the migration from RESTEasy to GitLab4j. GitLab4j suppors both V4 and V3 (even version 6.0.0.rc1). The RESTEasy implementation supports both V4 and V3. Dropping support for V3 on the grounds that GitLab < 11 is too old is a completely valid thing to do, but it has nothing to do with the project to migrate to GitLab4j. If doing this is desired, it should be done in a _separate_ PR targeting the main branch as described in https://github.com/jenkinsci/gitlab-plugin/pull/1491/files#r1210826532.